tuna/miopen/worker/fin_class.py

```diff
@@ -142,7 +142,7 @@ def set_db_tables(self):
                               config_type=self.config_type)
 
   def compose_work_objs(self, session, conds):
```

Comment:
can you add type hints for this function? and other functions you have touched. Also update the Jenkinsfile to call mypy on these files.